src_i18n/081_close_position.ts

```diff
@@ -40,189 +32,21 @@ async function main() {
-    //LANG:JP TokenExtensions のトークン情報を取得
-    //LANG:EN Get token info for TokenExtensions
-    tokenExtensionCtx: await TokenExtensionUtil.buildTokenExtensionContext(ctx.fetcher, whirlpool_data),
-  });
-
-  //LANG:JP 見積もり結果表示
-  //LANG:EN Output the estimation
-  console.log("devSAMO min output:", DecimalUtil.fromBN(quote.tokenMinA, token_a.decimals).toFixed(token_a.decimals));
-  console.log("devUSDC min output:", DecimalUtil.fromBN(quote.tokenMinB, token_b.decimals).toFixed(token_b.decimals));
-  
-  //LANG:JP 流動性を引き出す命令を作成
-  //LANG:EN Build the instruction to decrease liquidity
-  const decrease_liquidity_ix = WhirlpoolIx.decreaseLiquidityIx(
-    ctx.program,
-    {
-      ...quote,
-      whirlpool: whirlpool_pubkey,
-      position: position_pubkey,
-      positionAuthority: position_owner,
-      positionTokenAccount: position_token_account,
-      tokenOwnerAccountA: token_account_map.get(token_a.mint.toBase58()),
-      tokenOwnerAccountB: token_account_map.get(token_b.mint.toBase58()),
-      tokenVaultA: whirlpool.getData().tokenVaultA, 
-      tokenVaultB: whirlpool.getData().tokenVaultB,
-      tickArrayLower: tick_array_lower_pubkey,
-      tickArrayUpper: tick_array_upper_pubkey,
-    }
-  );  
-
-  //LANG:JP クローズ・ポジション命令を作成
-  //LANG:EN Build the instruction to close the position
-  const close_position_ix = WhirlpoolIx.closePositionIx(
-    ctx.program,
-    {
-      position: position_pubkey,
-      positionAuthority: position_owner,
-      positionTokenAccount: position_token_account,
-      positionMint: position.getData().positionMint,
-      receiver: position_owner,
-    }
-  );
-
-  //LANG:JP トランザクション組み立て
-  //LANG:EN Create a transaction and add the instruction
-  const tx_builder = new TransactionBuilder(ctx.connection, ctx.wallet);
-  //LANG:JP トークンアカウント作成
-  //LANG:EN Create token accounts
-  required_ta_ix.map((ix) => tx_builder.addInstruction(ix));
-  tx_builder
-    //LANG:JP 更新&フィー回収&リワード回収
-    //LANG:EN Update fees and rewards, collect fees, and collect rewards
-    .addInstruction(update_fee_and_rewards_ix)
-    .addInstruction(collect_fees_ix)
-    .addInstruction(collect_reward_ix[0])
-    .addInstruction(collect_reward_ix[1])
-    .addInstruction(collect_reward_ix[2])
-    //LANG:JP 流動性引き出し
-    //LANG:EN Decrease liquidity
-    .addInstruction(decrease_liquidity_ix)
-    //LANG:JP クローズ・ポジション
-    //LANG:EN Close the position
-    .addInstruction(close_position_ix);
+  const close_position_tx = await whirlpool.closePosition(position_pubkey, slippage);
 
   //LANG:JP トランザクションを送信
   //LANG:EN Send the transaction
-  const signature = await tx_builder.buildAndExecute();
-  console.log("signature:", signature);
-
-  //LANG:JP トランザクション完了待ち
-  //LANG:EN Wait for the transaction to complete
-  const latest_blockhash = await ctx.connection.getLatestBlockhash();
-  await ctx.connection.confirmTransaction({signature, ...latest_blockhash}, "confirmed");
+  for (const tx of close_position_tx) {
```

Comment:
Could you add the following comment here ?
```
//LANG:JP Close は手数料やリワードの回収など必要な処理を多く含むため、2つ以上のトランザクションに分割される場合があります。
//LANG:JP ただし、多くの場合は1つのトランザクションであることを期待できます。
//LANG:EN Since Close includes various necessary processes such as fee and reward collection, it may be split into two or more transactions.
//LANG:EN However, in most cases, you can expect it to be a single transaction.
```